lib/test/Test/Pos/Cbor/CborSpec.hs

```diff
@@ -71,124 +66,22 @@ import           Test.Pos.Util.QuickCheck (SmallGenerator)
-                          Bi aa, Bi ab, Eq aa, Arbitrary a, Show aa)
-    => Property
-soundSerializationAttributesOfAsProperty = forAll arbitraryAttrs $ \input ->
-    let serialized      = serialize input
-        (middle  :: ab) = unsafeDeserialize serialized
-        (encoded :: aa) = unsafeDeserialize $ serialize middle
-    in encoded === input
-  where
-    arbitraryAttrs :: Gen aa
-    arbitraryAttrs = Attributes <$> arbitrary <*> arbitrary
-
-
 spec :: Spec
 spec = withDefConfiguration $ \_ -> do
     describe "Cbor.Bi instances" $ do
         modifyMaxSuccess (const 1000) $ do
-            describe "Test instances" $ do
-                binaryTest @MyScript
-                prop "X2" (soundSerializationAttributesOfAsProperty @X2 @X1)
             describe "Lib/core instances" $ do
-                binaryTest @(Attributes X1)
-                binaryTest @(Attributes X2)
                 brokenDisabled $ binaryTest @UserSecret
                 modifyMaxSuccess (min 50) $ do
                     binaryTest @WalletUserSecret
                     binaryTest @EncryptedSecretKey
 
 
         describe "Types" $ do
-          -- 100 is not enough to catch some bugs (e.g. there was a bug with
-          -- addresses that only manifested when address's CRC started with 0x00)
-          describe "Bi instances" $ do
-              describe "Core.Address" $ do
-                  binaryTest @T.Address
-                  binaryTest @T.Address'
-                  binaryTest @T.AddrType
-                  binaryTest @T.AddrStakeDistribution
-                  binaryTest @T.AddrSpendingData
-              describe "Core.Types" $ do
-                  binaryTest @T.Timestamp
-                  binaryTest @T.TimeDiff
-                  binaryTest @T.EpochIndex
-                  binaryTest @T.Coin
-                  binaryTest @T.CoinPortion
-                  binaryTest @T.LocalSlotIndex
-                  binaryTest @T.SlotId
-                  binaryTest @T.EpochOrSlot
-                  binaryTest @T.SharedSeed
-                  binaryTest @T.ChainDifficulty
-                  binaryTest @T.SoftforkRule
-                  binaryTest @T.BlockVersionData
-                  binaryTest @(Attributes ())
-                  binaryTest @(Attributes T.AddrAttributes)
-              describe "Core.Fee" $ do
-                  binaryTest @T.Coeff
-                  binaryTest @T.TxSizeLinear
-                  binaryTest @T.TxFeePolicy
-              describe "Core.Script" $ do
-                  binaryTest @T.Script
-              describe "Core.Vss" $ do
-                  binaryTest @T.VssCertificate
-              describe "Core.Version" $ do
-                  binaryTest @T.ApplicationName
-                  binaryTest @T.SoftwareVersion
-                  binaryTest @T.BlockVersion
-              describe "Util" $ do
-                  binaryTest @(NewestFirst NE U)
-                  binaryTest @(OldestFirst NE U)
```

Comment:
These tests for `NewestFirst` and `OldestFirst` were duplicates of tests that had already been moved to `core/test/Test/Pos/Core/ChronoSpec.hs`